justfile

```diff
@@ -0,0 +1,95 @@
+alias b := build
+alias l := lint
+alias t := test
+
+examples := "deno-fresh express faas-experimental fastify hono-bun"
+
+test_permissions := "--allow-env --allow-net --allow-read --allow-sys"
+
+build: (build-npm "beta")
+
+build-npm version:
```

Comment:
this sets the same version for all packages - however, we often maintain and release the packages separately (for fixes and improvements)

Reply:
Hm, I don't think that's necessarily a problem, since this only *generates* packages, it doesn't publish them. so I think we can solve that by making sure we can specify in the CI job which packages actually need to be published, right? Most commonly you would just publish all, but for fixes, you just specify a specific package and its version.

Reply:
yeah as long as we have a way to publish them separately I think that's fine - I think a specification in CI works

Reply:
Since it's already like this in `main` I'll keep this suggestion for a separate PR.